src/IB/IBFEMethod.cpp

```diff
@@ -1707,7 +1707,7 @@ IBFEMethod::addWorkloadEstimate(Pointer<PatchHierarchy<NDIM> > hierarchy, const
 
         std::vector<double> workload_per_processor(n_processes);
         HierarchyCellDataOpsReal<NDIM, double> hier_cc_data_ops(hierarchy);
-        workload_per_processor[current_rank] = hier_cc_data_ops.L1Norm(d_workload_idx, -1, true);
+        workload_per_processor[current_rank] = hier_cc_data_ops.L1Norm(workload_data_idx, -1, true);
```

Comment:
`IBTK::invalid_index` instead of `-1`?
```suggestion
        workload_per_processor[current_rank] = hier_cc_data_ops.L1Norm(workload_data_idx, invalid_index, true);
```

Reply:
Since this is not an IBTK object let me explicitly write `IBTK::invalid_index`, but yes.

Reply:
I force pushed a fix.